spec/src/main/asciidoc/architecture.adoc

```diff
@@ -282,11 +282,22 @@ how such application servers should behave if they want to support MicroProfile
 Metrics from all applications and scopes should be available under a single REST endpoint ending with `/metrics` similarly as
 in case of single-application deployments (microservices).
 
-To help distinguish between metrics pertaining to each deployed application, a tag named `mp_app` should be added to each metric. 
+To help distinguish between metrics pertaining to each deployed application, a tag named `mp_app` should be added to each metric.
 
-The value of the `mp_app` tag should be passed by the application server to the application via a MicroProfile Config property named `mp.metrics.appName`.
-It should be possible to override this value by bundling the file `META-INF/microprofile-config.properties` within the application archive
-and setting a custom value for the property `mp.metrics.appName` inside it.
+The value of the `mp_app` tag should be passed by the application server to the application via a MicroProfile Config property named `mp.metrics.appName`. The property is to be defined at the application level through the `META-INF/microprofile-config.properties` file within the application archive.
```

Comment:
after this paragraph, we need to clearly define what an impl must do with the mp.metrics.appName config value.